pretix_eth/payment.py

```diff
@@ -30,20 +31,21 @@ def settings_content_render(self, request):
 
     @property
     def settings_form_fields(self):
-        d = OrderedDict(list(super().settings_form_fields.items()) +
-                        [
-                            ('ETH', forms.CharField(
-                                label=_('Ethereum wallet address'),
-                                help_text=_('Leave empty if you do not want to accept ethereum.'),
-                                required=False
-                            )),
-                            ('DAI', forms.CharField(
-                                label=_('DAI wallet address'),
-                                help_text=_('Leave empty if you do not want to accept DAI.'),
-                                required=False
-                            ))
-                        ]
-                        )
+        d = OrderedDict(
+            list(super().settings_form_fields.items())
+            + [
+                ('ETH', forms.CharField(
+                    label=_('Ethereum wallet address'),
+                    help_text=_('Leave empty if you do not want to accept ethereum.'),
```

Comment:
Maybe some context from the ui would help - but these two `help_text`s are not fun to mentally parse, especially when dealing with payments

Reply:
I'm gonna leave the UX stuff like this out for now.  Around the end of the week we should be able to address this stuff on a live "testing" site and see what sort of language changes might improvet he UX.

Reply:
A less double-negative-y option:
> Leave empty to disable payments in Ether